core/commonui/lemmyui/src/commonMain/kotlin/com/github/diegoberaldin/raccoonforlemmy/core/commonui/lemmyui/CommentCard.kt

```diff
@@ -176,14 +179,24 @@ fun CommentCard(
                             PostCardBody(
                                 text = comment.text.orEmpty(),
                                 autoLoadImages = autoLoadImages,
-                                onClick = onClick,
+                                onClick = {
+                                    if (textSelection.value) {
```

Comment:
if the previous change is applied this becomes as simple as

```suggestion
                                    if (textSelection) {
```